### ISSUE TITLE
Please make the "label"-prop required

### ISSUE THREAD

**Issue by ghost** [BosNaufal/vue2-autocomplete#54]
Hello,

I've debugged some time now the error 

> TypeError: Cannot read property 'split' of undefined

and basically found out that I was missing the "label"-prop. 
To prevent others from wasting time please make this a required prop since it is indeed required :)

Thank you!

**Comment by MichaelJPDX** [comment]
Yes, either make it required, or better yet remove the dependence on it that makes the module not work without it.

**Comment by yordivd** [comment]
+1
